hoot-core/src/main/cpp/hoot/core/elements/Node.cpp

```diff
@@ -104,17 +106,18 @@ boost::shared_ptr<geos::geom::Point> Node::toPoint() const
 
 QString Node::toString() const
 {
-  return
-    QString(
-      "Node(%1): x: %2 y: %3 tags:\n%4\n version: %5\n visible: %6\n status: %7\n circular error: %8")
-      .arg(getId())
-      .arg(QString::number(getX(), 'f', ConfigOptions().getWriterPrecision()))
-      .arg(QString::number(getY(), 'f', ConfigOptions().getWriterPrecision()))
-      .arg(getTags().toString())
-      .arg(getVersion())
-      .arg(getVisible())
-      .arg(getStatus().toString())
-      .arg(QString::number(getCircularError()));
+  stringstream ss(stringstream::out);
+  ss << "Node(" << getId() << "):"
+     << " x: " << QString::number(getX(), 'f', ConfigOptions().getWriterPrecision())
+     << " y: " << QString::number(getY(), 'f', ConfigOptions().getWriterPrecision())
+     << " tags:" << endl
+     << getTags().toString() << endl
+     << " version: " << getVersion() << endl
+     << " visible: " << getVisible() << endl
+     << " status: " << getStatus().toString();
+  if (hasCircularError())
+    ss << endl << " circular error: " << QString::number(getCircularError());
+  return QString::fromUtf8(ss.str().data());
```

Comment:
Make this similar to way and relation and only output `circular error` if one exists on the node and isn't just the default value.